presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -147,6 +148,15 @@ std::string nodeState2String(NodeState nodeState) {
   return fmt::format("<unknown>:{}>", static_cast<int>(nodeState));
 }
 
+std::string getRequestBody(const std::vector<std::unique_ptr<folly::IOBuf>>& body) {
```

Comment:
@czentgr asked me to move the code logic inside a function.

Reply:
I clarified with @czentgr.
We should put the whole block in a function say `handleGracefulShutdown(body)` similar to other `registerGet.. {server->reportNodeStatus(downstream)}` implementations. I agree that this will make it more readable.
```
      "/v1/info/state",
      [server = this](
          proxygen::HTTPMessage* /*message*/,
          const std::vector<std::unique_ptr<folly::IOBuf>>& body,
          proxygen::ResponseHandler* /*downstream*/) {
        server->handleGracefulShutdown(body);
      });
```

Reply:
Ok, I will make the changes.

Reply:
@majetideepak I have pushed new changes.